aws_common/src/sdk_utils/auth/service_credentials_provider.cpp

```diff
@@ -310,17 +314,16 @@ class RequestContext
 // IotRoleCredentialsProvider Class
 //
 
-IotRoleCredentialsProvider::IotRoleCredentialsProvider(const IotRoleConfig & config)
+IotRoleCredentialsProvider::IotRoleCredentialsProvider(const IotRoleConfig & config) : expiry_(0.0)
```

Comment:
Moving only one of them to the initializer list looks weird

Reply:
Done.